.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -2,30 +2,83 @@
 name: Bug report
 about: Create a report to help us improve
 title: ''
-labels: ''
+labels: 'bug'
 assignees: ''
 
 ---
 
-**Describe the bug**
-A clear and concise description of what the bug is.
-Please provide a full traceback of the error that you received (if any), regardless of how long it is. The full stack trace is important for us to be able to help you.
+<!--
+Please fill this template entirely and do not erase any of it.
+We reserve the right to close without a response bug reports which are incomplete.
```

Comment:
Is it worth saying that if you can't fill in the checklist, then it's likely that this is a question, not a bug, and probably belongs on discourse instead of here, where we track bugs and feature requests?

Reply:
That's a good point, I'll add that

Reply:
Should we just remove the "question" template at this point?

Reply:
I'm fine with removing the question template, though we probably should be a bit more proactive about questions on discourse if we do that.  I try to be responsive there, but I think I'm the only one.

It would be very nice if github issues were restricted to bugs and feature requests.

Reply:
Will do. I didn't realize how much activity the forum gets now. I'll subscribe and try to help out from now on.